src/configuration/structures/structures.ts

```diff
@@ -48,10 +48,17 @@ export interface BotFields {
   timezone?: string;
 }
 
+export interface BotTemplateFields {
+  name?: string;
+  avatar?: string;
+  default_group_priority?: GroupPriority;
+  job_title?: string;
```

Comment:
```suggestion
  name?: string;
  avatar?: string;
  default_group_priority?: GroupPriority;
  job_title?: string;
  max_chats_count?: number;
  affect_existing_installations?: bool
```
Should we add `max_chats_count` and `affect_existing_installations` properties here just like in lc-sdk-go?

https://github.com/livechat/lc-sdk-go/blob/b81f8a540508e0a86cb3834e23bafcfd5f76a14f/configuration/structures.go#L163-L170